egg/core/trainers.py

```diff
@@ -163,7 +163,8 @@ def eval(self):
         with torch.no_grad():
             for batch in self.validation_data:
                 batch = move_to(batch, self.device)
-                optimized_loss, interaction = self.game(*batch)
+                args, kwargs = extract_kwargs(batch)
```

Comment:
Are you assuming/checking kwargs is a dict of tensors? Otherwise this would fail without any catch https://github.com/facebookresearch/EGG/blob/master/egg/core/util.py#L320-L322

Reply:
I meant the move_to() call